profile/README.md

```diff
@@ -0,0 +1,138 @@
+            <img src="https://github.com/devicons/devicon/blob/master/icons/javascript/javascript-original.svg" alt="Javascript" height="50">
+        </a>
+        <a href="https://www.python.org/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/python/python-original.svg" alt="Python" height="50">
+        </a>
+        <a href="https://www.djangoproject.com/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/django/django-plain.svg" alt="Django" height="50">
+        </a>
+        <a href="https://fr.reactjs.org/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/react/react-original.svg" alt="React Js" height="50">
+        </a>
+        <a href="https://vuejs.org/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/vuejs/vuejs-original.svg" alt="Vue Js" height="50">
+        </a>
+        <a href="https://www.w3.org/html/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/html5/html5-original.svg" alt="HTML" height="50">
+        </a>
+        <a href="https://www.w3.org/Style/CSS/Overview.en.html">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/css3/css3-original.svg" alt="CSS" height="50">
+        </a>
+        <a href="https://www.mysql.com/fr/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/mysql/mysql-original.svg" alt="MySQL" height="50">
+        </a>
+        <a href="https://mui.com/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/materialui/materialui-plain.svg" alt="Mui" height="50">
+        </a>
+        <a href="https://azure.microsoft.com/fr-fr">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/azure/azure-original.svg" alt="Azure" height="50">
+        </a>
+        <a href="https://www.terraform.io/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/terraform/terraform-original.svg" alt="Terraform" height="50">
+        </a>
+        <a href="https://eslint.org/">
+            <img src="https://github.com/devicons/devicon/blob/master/icons/eslint/eslint-original.svg" alt="ESLint" height="50">
+        </a>
+    </details>
+</div>
+
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Puff.svg" alt="Correlatio logo" height="50"/>
+    <details>
+        <summary>
+            <h3>
+                Correlatio
+            </h3>
+        </summary>
+        <div align="left">
+            <p>
+                Correlatio offer large amount of statistical computation using AI based on open data of French government and other open statistics foundation.
+            </p>
+        </div>
+        <a href="">
+            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Crimen.svg" alt="Crimen logo" height="50">
```

Comment:
We should hold a repo for all of our logos, this way if we need to update one or replace a temporary placeholder we only would need one change.

Reply:
Yes, I think it's a good thing. In which `organization` I create it ?

Reply:
Well, I think the best option would be in the e-constellatio one. I do not see anywhere else to put it.